### ISSUE TITLE
Support arbitrary remark tags in embedded remarks

### ISSUE THREAD

**Issue by ronaldtse** [lutaml/expressir#78]
Due to metanorma/annotated-express#38, we need to support remark tags that are not part of an EXPRESS object. 

For example:
```
(*”action_schema.__fund_cons” ... *)
```

Should be iteratively made available and it’s contents made extractable from Expressir. 

This may mean a change in the assumption that an EXPRESS object only has one tagged remark.

**Comment by zakjan** [comment]
agreed approach:

generalize current InformalProposition to RemarkItem, because it can be a remark container of any kind https://github.com/lutaml/expressir/blob/master/lib/expressir/model/informal_proposition.rb
add remark_items to every model class that already has remarks, i.e. Identifier mixin https://github.com/lutaml/expressir/blob/master/lib/expressir/model/identifier.rb

source code

```
ENTITY test_entity; END_ENTITY;
(*"test_entity" AAA *)
(*"test_entity" BBB *)
(*"test_entity.IP1" CCC *)
(*"test_entity.IP1" DDD *)
(*"test_entity.anything" EEE *)
(*"test_entity.anything" FFF *)
```

parse tree

```
_class: Expressir::Model::Entity
id: test_entity
remarks:
- AAA
- BBB
informal_propositions:
  - _class: Expressir::Model::RemarkItem
    id: IP1
    remarks:
    - CCC
    - DDD
- remark_items:
  - _class: Expressir::Model::RemarkItem
    id: anything
    remarks:
    - EEE
    - FFF
```

usage - look it up by id

**Comment by zakjan** [comment]
Merged and releasing as 0.2.24
